packages/lesswrong/components/sequences/SequencesSmallPostLink.tsx

```diff
@@ -39,7 +40,11 @@ const SequencesSmallPostLink = ({classes, post, sequenceId}: {
 }) => {
   const { LWTooltip, PostsPreviewTooltip } = Components
 
-  const icon = !!post.lastVisitedAt ? <CheckBoxTwoToneIcon className={classes.read} /> : <CheckBoxOutlineBlankIcon className={classes.unread}/>
+  const { postsRead: clientPostsRead } = useItemsRead();
+
+  const isPostRead = !!post.lastVisitedAt || clientPostsRead[post._id];
```

Comment:
Not sure about the original motivation for using `!!post.lastVisitedAt` rather than `post.isRead` - I'm guessing they're both updated the first time a post is read?  For consistency's sake it'd probably be better to use `post.isRead` here, just want to make sure there isn't a reason not to.